core/bin/sqlite_worker.rs

```diff
@@ -214,198 +213,19 @@ async fn db_query(
     }
 }
 
-async fn delete_database_rows(
```

Comment:
All rows management endpoints are being deleted because now core uses the `databases_store` directly